src/script/page/MainContent/panels/preferences/avPreferences/DeviceSelect.tsx

```diff
@@ -52,27 +55,24 @@ const DeviceSelect: React.FC<DeviceSelectProps> = ({
       <div className="preferences-option-icon preferences-av-select-icon">
         <DeviceIcon />
       </div>
-      <div className="input-select">
-        <label className="label-medium" htmlFor={uieName}>
-          {title}
-        </label>
-        <select
-          className={cx('preferences-av-select', {'preferences-av-select-disabled': disabled})}
-          name="select"
-          disabled={disabled}
-          value={value}
-          data-uie-name={uieName}
+      <div
+        css={{
+          width: '550px',
+        }}
+      >
+        <Select
           id={uieName}
-          onChange={({target}) => onChange(target.value)}
-          aria-label={title}
-        >
-          {devices.map(({deviceId, label}) => (
-            <option key={deviceId} value={deviceId}>
-              {label || defaultDeviceName}
-            </option>
-          ))}
-        </select>
-        {!lessThanTwoDevices && <span className="icon-down preferences-av-label" />}
+          onChange={(selectedDevice: string) => {
```

Comment:
also I think it's better to extract this to a separate function, maybe `onSelectNewDevice`